test/functional/p2p_filter.py

```diff
@@ -64,7 +67,13 @@ def run_test(self):
 
         self.log.info('Check that too large filter is rejected')
         with self.nodes[0].assert_debug_log(['Misbehaving']):
-            filter_node.send_and_ping(msg_filterload(data=b'\xaa', nHashFuncs=51, nTweak=0, nFlags=1))
+            filter_node.send_and_ping(msg_filterload(data=b'\xaa', nHashFuncs=MAX_BLOOM_HASH_FUNCS+1, nTweak=0, nFlags=1))
+        with self.nodes[0].assert_debug_log(['Misbehaving']):
+            filter_node.send_and_ping(msg_filterload(data=b'\xbb'*(MAX_BLOOM_FILTER_SIZE+1)))
```

Comment:
Optional nit (please feel free to ignore): Would it make sense to either pass in `nHashFuncs=1` or change the default `nHashFuncs` from 0 to 1?

Reply:
Thanks for your review! I think it's fine to keep the default parameters of the msg_filterload constructor as they are -- they are in line with the equivalent of the `CBloomFilter()` constructor:

https://github.com/bitcoin/bitcoin/blob/fc00e651e407200da1eb0ae61f6448e22d1b6d8d/src/bloom.h#L67

What I did though was removing the explicit parameters for the first test case, as `nTweak` and `nFlags` are irrelevant for filters that won't even get accepted by the node, hence setting them doesn't improve the test.